src/dealDashboard/dealDashboard.scss

```diff
@@ -12,15 +12,14 @@
     "clauses aside"
     "discussion discussion"
     "thread thread";
-  gap: 0 115px;
+  gap: 60px 115px;
 
   deal-menubar {
     grid-area: navbar;
   }
 
   .top {
     grid-area: top;
-    margin-bottom: 40px;
 
     .title {
       padding-bottom: 12px;
```

Comment:
If a title is too long to fit in view, I suggest breaking it.
```suggestion
      padding-bottom: 12px;
      word-break: break-all;
```

### Current:
![image](https://user-images.githubusercontent.com/2517870/162012654-ca678b94-ac01-4ff3-b68c-9b973379eb0f.png)
### Expected:
![image](https://user-images.githubusercontent.com/2517870/162013259-b624e18b-4169-44b5-a74d-621076e9fe5a.png)